### DISCUSSION TITLE
Use 0s (or Xs) for placeholder phone numbers

### DISCUSSION THREAD

**Discussion by tbarmann** [jackocnr/intl-tel-input#1867] [Q&A] [answered]
The generated placeholder numbers look like actual phone numbers and can be confusing to users. Is there a way to change the numbers used so that they use only 0s or Xs so that it is clear this is a placeholder value and not a real phone number?  For example: (xxx) xxx-xxxx

**Comment by jackocnr** [comment, marked as answer]
This is possible with the `customPlaceholder` option, which is passed the generated placeholder number, so you can just replace all of the numeric digits however you like.

**Reply by tbarmann** [reply]
Perfect. Thanks!
